docs/reference/model_configuration.md

```diff
@@ -48,6 +48,16 @@ The SQLMesh project-level `model_defaults` key supports the following options, d
 - session_properties (on per key basis)
 - on_destructive_change (described [below](#incremental-models))
 
+
+### Model Naming
+
+Configuration option for name inference. Learn more in the [model naming guide](../guides/configuration.md#model-naming).
+
+| Option          | Description                                                                             |  Type   | Required |
+| --------------- | --------------------------------------------------------------------------------------- | :-----: | :------: |
+| `infer_names`   | Whether to automatically infer model names based on the directory structure (Default: true) | boolean |    N     |
```

Comment:
Is this table going to be rendered correctly?

Reply:
Yes though the default and type were wrong I corrected them